packages/public/client/src/main/styles/_content.scss

```diff
@@ -103,7 +103,8 @@
 
 // Typography
 
-.r {
+.r,
+.unrevised {
```

Comment:
Do you have a better idea? This is used at https://github.com/serlo/serlo.org/pull/151/files#diff-d3b1ffd861037a6291fafa3766d37d32R23 for a better styling of the h2 tags.

Reply:
I think we want to change this to `#content-layout`, though we would need to remove the `h1` then if we want the same look as right now. Don't think this is used anyway, because the editor content (rendered into `.r`) should never include an `h1`